package.json

```diff
@@ -56,7 +61,8 @@
     ],
     "files": [
       "src/**/*.test.ts"
-    ]
+    ],
+    "serial": true
```

Comment:
when using sinon.stub's on an async function like `client.post` it's important to run test serially